gapic/src/main/com/google/gapid/image/Images.java

```diff
@@ -276,6 +305,9 @@ public static Format from(Image.Format format) {
       boolean color = isColorFormat(format);
       int channels = getChannelCount(format, color ? COLOR_CHANNELS : DEPTH_CHANNELS);
       boolean is8bit = are8BitsEnough(format, color ? COLOR_CHANNELS : DEPTH_CHANNELS);
+      if (isCountFormat(format) && is8bit) {
```

Comment:
`is8bit` is still computed by only looking at the color or depth channels

Reply:
Done